src/parser.ts

```diff
@@ -2321,7 +2321,7 @@ function parseImportSpecifierOrNamedImports(
       // by the keyword 'as'.
       // See the ImportSpecifier production in ES6 section 15.2.2.
       validateBindingIdentifier(parser, context, BindingKind.Const, token, 0);
-      local = imported;
+      local = parser.cloneIdentifier(imported);
```

Comment:
Should this be under `options.uniqueKeyInPattern`?

Reply:
I guess so. But you are going to remove the option, so it's fine.